src/socket.c

```diff
@@ -3334,7 +3465,7 @@ static int handle_socket_input(const char *simbuffer, int simlen)
                 continue;  /* avoid non-telnet processing */
 
             } else if (xsock->fsastate == TN_SB) {
-		if (xsock->subbuffer->len > 30*1023) {
+		if (xsock->subbuffer->len > 255) {
```

Comment:
Why is that so lowered?

Reply:
Should be fine to raise it, though I haven't tested with a higher buffer.

Reply:
Changed this back, though keep an eye on it as it's significantly higher than I've found anyone else to be using. It might be good to put it in a #define or otherwise have some comment on the size and what is intended.